arches_her/webpack/webpack.common.js

```diff
@@ -243,6 +239,19 @@ module.exports = () => {
                     ],
                 },
             });
+        };
+
+        let projectSettings = spawn(
```

Comment:
ah -- this pattern has been updated since we last spoke

```
        let projectSettings = spawn(
            'python3',
            [Path.resolve(__dirname, Path.parse(__dirname)['dir'], 'settings.py')]
        );
        projectSettings.stderr.on("data", process.stderr.write);
        projectSettings.stdout.on("data", createWebpackConfig);

        projectSettings.on('error', () => {
            projectSettings = spawn(
                'python',
                [Path.resolve(__dirname, Path.parse(__dirname)['dir'], 'settings.py')]
            );
            projectSettings.stderr.on("data", process.stderr.write);
            projectSettings.stdout.on("data", createWebpackConfig);
        });
```